### PR TITLE
fix(curriculum): add links to Colab notebooks in TensorFlow course

### LINKED ISSUE #44790: TensorFlow course: Links to the Colab notebooks are missing
In this video, it is mentioned that "there is a link in the description", but I can't find it on the /learn platform.
https://www.freecodecamp.org/learn/machine-learning-with-python/tensorflow/core-learning-algorithms

When I searched on the forum I found this post:
https://forum.freecodecamp.org/t/tensorflow-course/492207
And I found the links in the description of the original video on YouTube.

I think we should include the links to the notebooks on /learn too.
Also, it may be better to mention that the original, not split video can be found there. Some people might prefer watching the not split version.

## Affected page

TensorFlow courses under Machine Learning with Python certification
https://www.freecodecamp.org/learn/machine-learning-with-python/

## Expected behavior

I'm not sure which page is the best place to put those links. We could put all of them at the first lesson, or it might be better to put each link to the beginning of the relevant section.

## System

- Device: Laptop
- OS: Windows 10
- Browser: chrome
- Version: 97.0.4692.71

### PR DESCRIPTION
**Pull request by praizjosh**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44790

Updated TensorFlow course challenges with missing colab notebook links.

I have tested the links and they work fine.

tensor-flow-course

### CODE REVIEW

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/convolutional-neural-networks.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ bilibiliIds:
 dashedName: convolutional-neural-networks
 ---
 
+# --description--
+
+<br>
+
+<small>The next few video lessons will reference [this Colab notebook](https://colab.research.google.com/drive/1ZZXnCjFEOkp_KdNcNabd14yok0BAIuwS#forceEdit=true&sandboxMode=true).</small>
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/convolutional-neural-networks.md:17]:
```suggestion
The next few video lessons will reference [this Colab notebook](https://colab.research.google.com/drive/1ZZXnCjFEOkp_KdNcNabd14yok0BAIuwS#forceEdit=true&sandboxMode=true).
```

We don't really need any of this extra HTML.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
You'll want to do the same for the other 5 challenges:

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/convolutional-neural-networks.md (hunk 1 above)
Comment (praizjosh) [reply curriculum/challenges/english/11-machine-learning-with-python/tensorflow/convolutional-neural-networks.md]: Sure. I will removed them. Only added them for legibility. Thanks.

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/convolutional-neural-networks.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ bilibiliIds:
 dashedName: convolutional-neural-networks
 ---
 
+# --description--
+
+---
+
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/convolutional-neural-networks.md:16]:
```suggestion
```

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/core-learning-algorithms.md [hunk 3]
```diff
@@ -10,6 +10,12 @@ bilibiliIds:
 dashedName: core-learning-algorithms
 ---
 
+# --description--
+
+---
+
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/core-learning-algorithms.md:16]:
```suggestion
```

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/introduction-to-tensorflow.md [hunk 4]
```diff
@@ -10,11 +10,17 @@ bilibiliIds:
 dashedName: introduction-to-tensorflow
 ---
 
+# --description--
+
+---
+
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/introduction-to-tensorflow.md:16]:
```suggestion
```

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/introduction-to-tensorflow.md [hunk 5]
```diff
@@ -10,11 +10,17 @@ bilibiliIds:
...
+The next few video lessons will reference [this Colab notebook](https://colab.research.google.com/drive/1F_EWVKa8rbMXi3_fG0w7AtcscFq7Hi7B#forceEdit=true&sandboxMode=true).
+
 # --question--
 
 ## --text--
 
-Which of the following is **not** a type of tensor?
+Which of the following is **not** a type of tensor?
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/introduction-to-tensorflow.md:23]:
```suggestion
Which of the following is **not** a type of tensor?
```

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/natural-language-processing-with-rnns.md [hunk 6]
```diff
@@ -10,6 +10,12 @@ bilibiliIds:
 dashedName: natural-language-processing-with-rnns
 ---
 
+# --description--
+
+---
+
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/natural-language-processing-with-rnns.md:16]:
```suggestion
```

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/neural-networks-with-tensorflow.md [hunk 7]
```diff
@@ -10,10 +10,17 @@ bilibiliIds:
 dashedName: neural-networks-with-tensorflow
 ---
 
+# --description--
+
+---
+
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/neural-networks-with-tensorflow.md:16]:
```suggestion
```

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/neural-networks-with-tensorflow.md [hunk 8]
```diff
@@ -10,10 +10,17 @@ bilibiliIds:
...
+The next few video lessons will reference [this Colab notebook](https://colab.research.google.com/drive/1m2cg3D1x3j5vrFc-Cu0gMvc48gWyCOuG#forceEdit=true&sandboxMode=true).
+
 # --question--
 
 ## --text--
 
+
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/neural-networks-with-tensorflow.md:23]:
```suggestion
```

File: curriculum/challenges/english/11-machine-learning-with-python/tensorflow/reinforcement-learning-with-q-learning.md [hunk 9]
```diff
@@ -10,6 +10,12 @@ bilibiliIds:
 dashedName: reinforcement-learning-with-q-learning
 ---
 
+# --description--
+
+---
+
```
Comment (moT01) [inline curriculum/challenges/english/11-machine-learning-with-python/tensorflow/reinforcement-learning-with-q-learning.md:16]:
```suggestion
```

**Review by moT01 — COMMENTED** [review]
I'm getting a little nit-picky with some of the spacing suggestions - but I would remove the three dashes in all the files as well. I know it doesn't look great without some spacing below the video, but I think that should be fixed in the view component instead of trying to add markdown or HTML in the description.

**Review by naomi-lgbt — APPROVED** [review]

**Review by huyenltnguyen — APPROVED** [review]

**Pull request merged**